spec/rubocop/cop/rspec/sort_metadata_spec.rb

```diff
@@ -23,6 +23,50 @@
     RUBY
   end
 
+  it 'registers an offense when a symbol metadata is before second docstring ' \
+     'argument' do
+    expect_offense(<<~RUBY)
+      RSpec.describe 'Something', :a, :z, 'second docstring' do
+                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^ Sort metadata alphabetically.
+      end
+    RUBY
+
+    expect_correction(<<~RUBY)
+      RSpec.describe 'Something', 'second docstring', :a, :z do
+      end
+    RUBY
+  end
+
+  it 'registers an offense when a symbol metadata is before a variable ' \
+     'argument' do
+    expect_offense(<<~RUBY)
+      RSpec.describe 'Something', :a, :z, variable, foo: :bar do
```

Comment:
If the `variable` holds something that is not a symbol, this will fail with a similar ArgumentError